packages/orbit-components/src/primitives/ButtonPrimitive/__tests__/index.test.js

```diff
@@ -75,4 +79,21 @@ describe("ButtonPrimitive", () => {
     );
     expect(screen.getByRole("link")).toHaveAttribute("href", "#");
   });
+
+  it("should behave as button", () => {
+    const children = "Lorem ipsum";
+    const dataTest = "test";
+
+    render(
+      <ButtonPrimitive dataTest={dataTest} tabIndex={0} asComponent="div">
+        {children}
+      </ButtonPrimitive>,
+    );
+
+    const button = screen.getByTestId(dataTest);
+
+    userEvent.tab();
+    fireEvent.keyDown(button);
```

Comment:
To check whether `onClick` has been called when <kbd>⮐</kbd> is pressed you need to pass the event object as the second argument (and the `onClick` mock to `ButtonPrimitive`).

Reply:
yes, you are right, thx, fixed